.travis.yml

```diff
@@ -3,9 +3,11 @@ dist: trusty
 language: java
 
 before_install:
+  - sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+  - wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
```

Comment:
Shouldn't matter. We don't own the hardware, and Travis is not the final arbiter of whether or not a change is merged.